activestorage/lib/active_storage/attached.rb

```diff
@@ -8,8 +8,12 @@ module ActiveStorage
   class Attached
     attr_reader :name, :record
 
-    def initialize(name, record)
-      @name, @record = name, record
+    def initialize(name, record, configuration)
+      @name, @record, @configuration = name, record, configuration
+    end
+
+    def find_variant_by_name(variant_name)
```

Comment:
I keep going back and forth on whether or not we should just call this (and the internal language) `transformations_for(key)` since they aren't full variants. Either case this should be `# :nodoc:` so it's private API.